dsl/src/test/kotlin/com/chutneytesting/kotlin/dsl/ChutneyFunctionsTest.kt

```diff
@@ -175,6 +175,41 @@ class ChutneyFunctionsTest {
         )
     }
 
+    @Test
+    fun `use jsonMerge function`() {
```

Comment:
You could add each new functions in **should wrap expression for evaluation by default`** test method.

Reply:
Fixed